src/Compilers/Core/Desktop/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -335,8 +335,16 @@ public IEnumerable<AnalyzerReference> ResolveAnalyzerReferences(IAnalyzerAssembl
 
         internal ImmutableArray<DiagnosticAnalyzer> ResolveAnalyzersFromArguments(string language, List<DiagnosticInfo> diagnostics, CommonMessageProvider messageProvider, TouchedFileLogger touchedFiles, IAnalyzerAssemblyLoader analyzerLoader)
```

Comment:
Is this expected to be snappy? Can this regress command line build? Are we reading the metadata twice from disk (once for consistency checking and later when the analyzers are actually loaded) or does the act of reading the metadata also load the binaries?